dashboard-parent/dashboard-core/src/main/java/org/wicketstuff/dashboard/AbstractWidget.java

```diff
@@ -31,6 +31,7 @@
 	protected WidgetLocation location;
 
 	public AbstractWidget() {
+		this.title = "";
```

Comment:
I would better rewrite `hashcode` and `equals`
it uses both title and id
Any IDE can provide failsafe implementation

Reply:
Added a eclipse generated version.